changelog/v1.15.0-beta19/bump-envoy-gloo.yaml

```
@@ -0,0 +1,10 @@
+changelog:
+- type: DEPENDENCY_BUMP
+  description: Bump envoy gloo to latest v1.25.8-patch1
+  dependencyOwner: solo-io
+  dependencyRepo: envoy-gloo
+  dependencyTag: v1.25.8-patch1
+- type: FIX
+  issueLink: https://github.com/solo-io/solo-projects/issues/5138
```

And you should probably remove the `BOT NOTES` section in the main PR description

```suggestion
  issueLink: https://github.com/solo-io/solo-projects/issues/5138
  resolvesIssue: false
```